thundra/reporter.py

```diff
@@ -60,7 +61,10 @@ def prepare_report_json(self):
             try:
                 report_jsons.append(json.dumps(report))
             except TypeError:
-                logger.error("Couldn't dump report with type {} to json string, \
-                            probably it contains a byte array".format(report.get('type'))) 
+                logger.error(("Couldn't dump report with type {} to json string, "
```

Comment:
There might be other cases rather than byte array. So it is better not to mention byte array in the error message